app/frontend/src/components/forms/FormSubmission.vue

```diff
@@ -87,16 +110,27 @@ export default {
       loading: true,
     };
   },
-  computed: mapGetters('form', ['form', 'formSubmission']),
+  computed: mapGetters('form', ['form', 'formSubmission', 'permissions']),
   methods: {
-    ...mapActions('form', ['fetchSubmission']),
+    ...mapActions('form', ['fetchSubmission', 'getFormPermissionsForUser']),
     refreshNotes() {
       this.$refs.notesPanel.getNotes();
     },
+
+    canViewAllSubmissions() {
+      const perms = [
+        FormPermissions.SUBMISSION_READ,
+        FormPermissions.SUBMISSION_UPDATE,
+      ];
+      return this.permissions.some((p) => perms.includes(p));
```

Comment:
Not sure if we should be blocking navigation from the view submission page to the submissions table. At this time, the /form/view page requires you to be authenticated according to the route, so public users won't be able to look at this page anyways and thus makes this check not relevant. However, if we do plan on making this view not authentication forced, then this permission check would be useful as a submitter user (non-staff) would never need to see this button nor know about it.

Reply:
If you're viewing a submission you're a reviewer so there's no way to get into that without having the ability to view submissions in the current model. So yeah being able to go from a submission to the submissions table is kind of a given.